### PR TITLE
Disclaimers and cookies pop ups

### PR DESCRIPTION
**Pull request by derickbailey**
This pull request turns the notice at the top of hte /packages page into a full-screen pop up notification, requiring you to acknowledge it before you can do anything else on the site.

It also adds a cookie policy pop-up to the bottom of the site, which can also be closed.

in both cases, closing the pop up will remain "closed" in that browser instance for up to 100 years, by setting a cookie to prevent the page from displaying the pop up.

any time the cookie is not present, the pop ups will show up again

### CODE REVIEW

File: chocolatey/Website/Website.csproj [hunk 1]
```diff
@@ -31,6 +31,7 @@
     <IISExpressAnonymousAuthentication />
     <IISExpressWindowsAuthentication />
     <IISExpressUseClassicPipelineMode />
+    <UseGlobalApplicationHostFile />
```
Comment (ferventcoder) [inline chocolatey/Website/Website.csproj]: I wonder where this snuck in from. No worries...

**Comment by ferventcoder** [comment]
Did you want to squash this down or do you want me to handle that?

File: chocolatey/Website/Website.csproj (hunk 1 above)
Comment (derickbailey) [reply chocolatey/Website/Website.csproj]: that's odd... maybe my specific version of visual studio introduced them for soem reason? i'll take these out and squash the commits.

**Comment by derickbailey** [comment]
alright, i took out that strange line in teh website.csproj and squashed the commits for the PR into my master branch

**Comment by ferventcoder** [comment]
I completely forgot to tell you about our super heavy caching strategies - any server side code checks won't work so most things like this would need to be done client side. I filed and implement #542, but it won't go back out until #543 gets addressed.

**Pull request merged**